src/DemoUI/src/Views/resources/css/index.css

```diff
@@ -0,0 +1,1865 @@
+    color: #5d5d5d;
+    font-size: 12px !important;
+    line-height: 14px !important;
+    font-weight: normal;
+    text-align: left !important;
+    border-radius: 3px;
+    border: 1px solid #dcdcdc;
+    z-index: 1;
+    visibility: hidden;
+    opacity: 0;
+    transition: visibility 0.3s linear, opacity 0.3s linear;
+}
+
+.pl-tooltip-left {
+    top: 50%;
+    transform: translateY(-50%);
+    bottom: auto;
+    left: -155px;
+}
+
+.pl-tooltip-left:before,
+.pl-tooltip-left:after {
+    content: '';
+    display: block;
+    position: absolute;
+    left: 100%;
+    width: 0;
+    height: 0;
+    border-style: solid;
+}
+
+.pl-tooltip-left:after {
+    top: calc(50% - 5px);
+    border-color: transparent transparent transparent white;
+    border-width: 6px;
+}
+
+.pl-tooltip-left:before {
+    top: calc(50% - 6px);
+    border-color: transparent transparent transparent #dcdcdc;
+    border-width: 7px;
+}
+
+.pl-form-section-subtitle-wrapper {
+    margin: 0 !important;
+    padding: 0 !important;
+    height: 20px;
+    line-height: 12px;
+    color: #5d5d5d;
+    font-size: 12px;
+}
+
+.pl-title-carrier-logo {
+    margin-top: 10px !important;
+}
+
+.pl-shipping-countries {
+    margin-top: 20px !important;
+}
+
+.pl-destination-countries-selector {
+    width: 100%;
+    border: 2px solid #dcdcdc;
+    padding: 10px;
+}
+
+.pl-destination-countries-selector > option{
+    font-size: 16px;
+    color: #5d5d5d;
+}
+
+.pl-form-tax {
+    margin-top: 10px !important;
+}
+
+.pl-form-section-input-wrapper {
+    margin: 0;
+    padding: 0;
+    color: #5d5d5d;
+}
+
+.pl-text-input-label {
+    height: 16px;
+    line-height: 7px;
+    position: absolute;
+    top: 22px;
+    left: 0;
+    margin-left: 15px;
+    text-transform: uppercase;
+    font-size: 17px;
+    font-weight: bold;
+    color: #9b9b9b;
+    pointer-events: none;
+}
+
+#pl-login-api-key-label {
+    line-height: 17px !important;
+    margin-left: 25px !important;
+}
+
+.pl-text-input-label.selected {
+    height: 12px;
+    line-height: 12px;
+    position: absolute;
+    top: 12px;
+    left: 0;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: bold;
+    color: #5d5d5d;
+    pointer-events: none;
+    transition: all .25s;
+}
+
+.pl-input-search-icon {
+    position: absolute;
+    top: 2px;
+    right: 0;
+    width: 50px;
+    height: 48px;
+    padding: 10px;
+    box-sizing: border-box;
+    border-left: 1px solid #dcdcdc;
+    cursor: pointer;
+}
+
+.pl-input-search-icon * {
+    pointer-events: none;
+    -webkit-touch-callout: none; /* iOS Safari */
+    -webkit-user-select: none; /* Safari */
+    -moz-user-select: none; /* Firefox */
+    -ms-user-select: none; /* Internet Explorer/Edge */
+    user-select: none;
+}
+
+.pl-form-section-input {
+    position: relative;
+    border: none;
+}
+
+.pl-form-section-input input {
+    height: 50px !important;
+    background-color: white !important;
+    border-radius: 0 !important;
```

Comment:
Codacy found an issue: [Use of !important](https://app.codacy.com/gh/packlink-dev/ecommerce_module_core/pullRequest?prid=5826400)

Reply:
This file is deleted (replaced) in the next PR.